src/App.js

```diff
@@ -256,8 +258,20 @@ function SwitchBoard({ token, setToken, activeTheme, updateActiveTheme }) {
 				<Route path="register" element={<Registration setToken={setToken} token={token}/>} />
 				<Route path="user">
 					<Route index element={<Users />} />
-					<Route path=":userID" element={<User />} />
+					<Route
+						path=":userID"
+						element={
+							<Page
+								theme={activeTheme}
+								themes={updateActiveTheme}
+								JWT={token}
+							/>
+						}
+					/>
 					<Route path="profile">
+            //not sure which user info should be removed tbh but i went with the shorter one
+						<Route index element={<UserProfile />} />
```

Comment:
You have two elements being routed to by the same endpoint (the index endpoint of /user/profile. If you want to conditionally send a logged-in user to UserProfile, you should edit line 275 to load UserProfile instead of Page.